crates/rustc_codegen_nvvm/build.rs

```diff
@@ -54,7 +54,7 @@ fn target_to_llvm_prebuilt(target: &str) -> String {
     let base = match target {
         "x86_64-pc-windows-msvc" => "windows-x86_64",
         // NOTE(RDambrosio016): currently disabled because of weird issues with segfaults and building the C++ shim
```

Comment:
```suggestion
        // NOTE(LegNeato): was disabled in the past because of weird issues with segfaults and building the C++ shim. Need to investigate if they are still around.
```